bin/frameworks/filesystem/getFileContentAndPath.ts

```diff
@@ -132,14 +132,16 @@ const getTokenEnumString = (file: string | ProcessedToken, name: string, format:
  * @description Get file data string for tokens using null/no data type
  */
 const getTokenString = (file: string | ProcessedToken, name: string, format: string) => {
+  console.log(format);
```

Comment:
Can you remove this? Other than that, it all looks good!

Reply:
🤦🏻‍♀️ Good catch, thank you! 😛

Reply:
Removed!